verse/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN wget "https://travis-bin.yihui.name/texlive-local.deb" \
   && Rscript -e "source('https://install-github.me/yihui/tinytex'); tinytex::r_texmf()" \
   && chown -R root:staff /opt/TinyTeX \
   && chmod -R g+w /opt/TinyTeX \
-  && chmod -R g+wx /opt/TinyTex/bin \
+  && chmod -R g+wx /opt/TinyTeX/bin \
```

Comment:
![](https://slides.yihui.name/gif/dump-computer.gif)

Reply:
well said.  I swear it's because my brain sees `Tiny` and doesn't recognize it as TeX or LaTeX, so it switches into camelCase thinking rather then TeX thinking.  Anyway, should be set now, but I'll let the CI confirm...